bindings/c/src/lib.rs

```diff
@@ -72,6 +72,41 @@ pub unsafe extern "C" fn opendal_operator_new(scheme: *const c_char) -> opendal_
     opendal_operator_ptr::from(op)
 }
 
+/// Free the allocated operator pointed by [`opendal_operator_ptr`]
+#[no_mangle]
+pub extern "C" fn opendal_operator_free(op_ptr: opendal_operator_ptr) {
+    let _ = unsafe { Box::from_raw(op_ptr.get_ref_mut()) };
+    // dropped
+}
+
+/// Get current operator's limit
+#[no_mangle]
+pub extern "C" fn opendal_operator_limit_get(op_ptr: opendal_operator_ptr) -> usize {
```

Comment:
Please don't expose those API before we have a working binding. Keep simple and force.

Reply:
No problem, then this PR should only be about the `opendal_operator_free`